pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -717,7 +716,9 @@ public String selectBrokerForAssignment(final ServiceUnitId serviceUnit) {
 
             LoadManagerShared.removeMostServicingBrokersForNamespace(serviceUnit.toString(), brokerCandidateCache,
                     brokerToNamespaceToBundleRange);
-            log.info("{} brokers being considered for assignment of {}", brokerCandidateCache.size(), bundle);
+            if (log.isDebugEnabled()) {
+                log.debug("{} brokers being considered for assignment of {}", brokerCandidateCache.size(), bundle);
```

Comment:
can we keep it info as LoadManager's logs are really helpful for various improvements.

Reply:
Sure